Index.html

```diff
@@ -5,11 +5,190 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
+    <link rel="stylesheet" href="/css/Main.css">
     <title>Profolio | Home</title>
 </head>
 <body>
-    <main>
+    <main class="Main_Container">
+        <nav>
+            <p>John Doe</p>
+            <i class="fas fa-bars"></i>
+        </nav>
```

Comment:
The container of this section should be a `header` tag rather than a `nav` tag 🧐 Kindly make the necessary adjustments to fulfill the requirements.